app/views/transaction/_additional_information_tabbed.html.erb

```diff
@@ -1,6 +1,6 @@
 <div class="js-tabs nav-tabs">
   <ul>
-    <% if transaction.more_information.present? %><li class="active"><a href="#more-information"><%= t 'formats.transaction.more_information' %></a></li><% end %>
+    <% if transaction.more_information.present? %><li class="active"><a href="#more-information"><%= t 'formats.transaction.before_you_start' %></a></li><% end %>
```

Comment:
There's an id here for 'more-information' should this be before you start? If so I guess we could update the ID